tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -105,32 +100,27 @@ drop_chunks_add_policy(PG_FUNCTION_ARGS)
 							get_rel_name(ht_oid))));
 		}
 
-		if (!DatumGetBool(DirectFunctionCall2(interval_eq,
-											  IntervalPGetDatum(&existing->fd.older_than),
-											  IntervalPGetDatum(older_than))) ||
-			(existing->fd.cascade != cascade) ||
-			(existing->fd.cascade_to_materializations != cascade_to_materializations))
+		if (ts_interval_equal(&existing->fd.older_than, older_than) &&
```

Comment:
is the not equal case ever hit here in tests?

Reply:
Both branches are hit in tsl_tables.sql tests
see [ sl_tables.out#L154 ](https://github.com/timescale/timescaledb/blob/2d8fe54d02b65b3e08d5f522bc54c2eab18b35dd/tsl/test/expected/tsl_tables.out#L154) for equal case
https://github.com/timescale/timescaledb/blob/2d8fe54d02b65b3e08d5f522bc54c2eab18b35dd/tsl/test/expected/tsl_tables.out#L154
see [ sl_tables.out#L162 ](https://github.com/timescale/timescaledb/blob/2d8fe54d02b65b3e08d5f522bc54c2eab18b35dd/tsl/test/expected/tsl_tables.out#L162) for not equal case
https://github.com/timescale/timescaledb/blob/2d8fe54d02b65b3e08d5f522bc54c2eab18b35dd/tsl/test/expected/tsl_tables.out#L162